runtime/src/configs/mod.rs

```diff
@@ -424,4 +424,5 @@ impl pallet_faucet::Config for Runtime {
 /// Config for our randomness pallet
 impl pallet_randomness::Config for Runtime {
     type AuthorVrfGetter = BabeDataGetter<Runtime>;
+    type WeightInfo = ();
```

Comment:
What does it do? Why are we setting weight info to nothing?

Reply:
We're not setting it to "nothing", WeightInfo is implemented for unit